getting-started/spark/create-polaris-catalog.sh

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN:-principal:root;realm:default-realm}"
+POLARIS_CATALOG_NAME="${POLARIS_CATALOG_NAME:-polaris_demo}"
+
+# create a catalog backed by the local filesystem
```

Comment:
I'm not entirely sure if we need this file. Could we handle everything directly within the notebook, like the other operations in `SparkPolaris.ipynb`? Would it simplify things if we moved the operations there?

Reply:
we could, but i think its a good idea to separate infra code (this script) from application code (the notebook)

Reply:
We could initialize catalog in notebook as well. I feel it's more flexible that way. for example, you don't have to worry about the an env variable for catalog name. But I'm OK with either one. Not a blocker for me.